### ISSUE TITLE
Protocol versioning

### ISSUE THREAD

**Issue by pvlugter** [cloudstateio/cloudstate#288]
Add cloudstate protocol versioning, so that we can align compatibility between the proxy and the language support modules. Ideally both sides can support multiple versions and negotiate on the latest supported.

**Comment by viktorklang** [comment]
There should already be versioning in the protocol: https://github.com/cloudstateio/cloudstate/blob/master/protocols/protocol/cloudstate/entity.proto#L172

**Comment by sleipnir** [comment]
yes it exists but is totally ignored by all supported languages, including java and node

**Comment by marcellanz** [comment]
No semantics are defined yet, we have to spec it! Lets work on #119 :)
I'd like to have a spec as promised by the project, a "A standards effort" (spec) and a "A reference implementation" \o/
